.github/workflows/nightly-build.yml

```diff
@@ -73,35 +73,40 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: oneapi-src/oneDAL
+      - name: Set MKL_FPK_GPU_VERSION_LINE
+        shell: bash
+        run: |
+          echo "MKL_FPK_GPU_VERSION_LINE=2024.0.0" >> "$GITHUB_ENV"
       - name: Install DPC++
         shell: cmd
         run: |
-            call .\.ci\scripts\install_windows.bat ${{ env.WINDOWS_BASEKIT_URL }} ${{ env.WINDOWS_ALL_COMPONENTS }}
+            call .\.ci\scripts\install_windows.bat ${{ env.WINDOWS_BASEKIT_URL }} ${{ env.WINDOWS_ALL_COMPONENTS }} .\oneapi\
```

Comment:
also, please update ci.yml file, because it also uses this install_windows.bat script

Reply:
ci.yml has been updated.